<APP_NAME>/health.py

```
@@ -0,0 +1,76 @@
+    response_headers: Dict[str, Any] = {}
+
+    try:
+        async def _run() -> httpx.Response:
+            async with httpx.AsyncClient(follow_redirects=True, verify=True) as client:
+                last_exc: Optional[Exception] = None
+                for attempt in range(retries + 1):
+                    try:
+                        resp = await _async_request(client, service.method, url, service.headers or {}, service.timeout_s)
+                        return resp
+                    except (httpx.ConnectError, httpx.ReadTimeout, httpx.ConnectTimeout, httpx.RemoteProtocolError, httpx.ReadError) as exc:
+                        last_exc = exc
```

Ruff (0.12.2)

Local variable `last_exc` is assigned to but never used

Remove assignment to unused variable `last_exc`

(F841)